demo/vl-map.html

```diff
@@ -45,6 +45,12 @@
                 <vl-map-baselayer-grb-gray></vl-map-baselayer-grb-gray>
             </vl-map>
         </vl-demo>
+
+        <vl-demo data-vl-title="Kaart met andere extent">
+            <vl-map id="map" data-vl-extent="[-73122.443418, -23915.714632, 347445.354629, 297624.297341]">
+                <vl-map-baselayer-grb-gray id="baselayer-grb-gray"></vl-map-baselayer-grb-gray>
+            </vl-map>
+        </vl-demo>
```

Comment:
Ik zie/merk geen verschil met de demo's zonder extend?

Reply:
Je kan verder panen in de map en verder uitzoomen